### ISSUE TITLE
Unable to reproduce the results with the pretrained models.

### ISSUE THREAD

**Issue by Ikarosy** [lwwu2/diver#10]
Hi,

I tried to reproduce the results with the pretrained models.  Following the [Reproduction](https://github.com/lwwu2/diver#reproduction) section did not help exactly.  Because I found the `mask_scale` is inconsistent within the same dataset.  For instance, for some senes in **Synthetic_NeRF** I had to set the `mask_scale` to 0.25, and set to 1.0 for others, so that I could solve the parameter mismatch [issue](https://github.com/lwwu2/diver/issues/9) reported before.  

The PSNR of Synthetic_NeRF with the released pretrained models with different `mask_scale`  is as genuine as reported in the paper.  But, in the case of **BlendedMVS**, the PSNR becomes really worse. Specifically, the PSNR is PSNR: 2.6886 in _Jade_ and 2.343 in _Fountain_.  I guess there should be something wrong with the configs/blendedmvs_fine.py.

In the case of **tanks&temples**, all scenes' reproduction results are acceptable except for _Truck_, whose PSNR is 12.93. 

So, basically the issues occur in reproducing _Jade_ and _Fountain_ of **Synthetic_NeRF** and _Truck_ of  **tanks&temples**. 

I wonder would it be possible for you to update the correct config files or the pertained checkpoints for the follow-up research of this renowned DIVeR?  It would be greatly appreciated for your fix-up!

Thank you in advance for the time!

Best regards,
Ikarosyyy

**Comment by lwwu2** [comment]
Hi,

I used a poorly written notebook script to convert checkpoints to the published training weight, where I stored the voxel_mask with 64 resolution for the Chair and the Fountain scene by mistake. It can be resolved by changing mask_scale = 0.25 as you suggested. 

For the same reason, I messed up with the coordinate conversion when storing the weight for the Truck scene, which I do not think can be fixed and I no longer have the original training weight. Therefore, you probably need to re-run the training code. Sorry for the inconvenience!

Because Jade and Fountain use black background color, the white background flag should be set to False in the config file as suggested [here](https://github.com/lwwu2/diver/blob/78684e5e5004896be9bc1cc3eaa01bb94f02c042/configs/blendedmvs_fine.py#L135), which should fix the issue.

Please let me know if you have more questions.

**Comment by Ikarosy** [comment]
Thanks for your reply!

**Comment by Ikarosy** [comment]
BTW, as a note for any follow-up fellows may query this issue.  
To evaluate this repository's official pertained models, 
`voxel_num` in any cases of config.py should be set 256 or 320, dependent on the specific checkpoint situation,
`mask_scale` should be set 0.25 or 1.0 according to the specific checkpoint situation,
`white_back` should be set False for jade and fountain of BlendedMVS.

Truck of tanks&temples is a broken checkpoint.  Except this one, all other pertained models work well as reported in the paper.

At last, great appreciation would be given to the author for this excellent work and persistent maintenance for this project!
